src/grpc/packet_router/hpr_packet_router_service.erl

```diff
@@ -69,6 +64,10 @@ handle_info({give_away, NewPid, PubKeyBin}, StreamState) ->
     lager:info("give_away registration to ~p", [NewPid]),
     gproc:give_away({n, l, ?REG_KEY(PubKeyBin)}, NewPid),
     StreamState;
+handle_info(?SESSION_RESET, StreamState0) ->
+    {EnvDown, StreamState1} = create_session_offer(StreamState0),
+    %% TODO: Do we reset the session key here?
```

Comment:
I'm not sure the order guarantees of bidirectional streams. 
But what I think this is asking is, when we send a new session offer, do we ignore any packets uplinked until the gw sends a new session key?
Probably a good gateway question, as they might change the order they need to process things in.

Reply:
> But what I think this is asking is, when we send a new session offer, do we ignore any packets uplinked until the gw sends a new session key?

You keep using the old/current session key (aka verifier key) until you get a new session init from the gateway